src/elements/content-sidebar/activity-feed/common/activity-message/messages.js

```diff
@@ -12,6 +12,16 @@ const messages = defineMessages({
         defaultMessage: 'Translate',
         description: 'Translate button for translating comment',
     },
+    activityMessageSeeMore: {
+        id: 'be.contentSidebar.activityFeed.common.seeMore',
+        defaultMessage: 'See more',
+        description: 'See more button for showing whole long message',
+    },
+    activityMessageSeeLess: {
+        id: 'be.contentSidebar.activityFeed.common.seeLess',
```

Comment:
minor: the `id`s for the messages above have a `Message` suffix

`id: 'be.contentSidebar.activityFeed.common.seeMoreMessage',`
`id: 'be.contentSidebar.activityFeed.common.seeLessMessage',`